Updating project profiles

LauraUSDS opened the pull request:
_Laura to double-check text, images, formatting on:_
- [x] each individual profile page
- [x] each project carousel card
- [x] full projects page

LauraUSDS committed:
create VA.gov profile

LauraUSDS committed:
update MPSM profile

LauraUSDS committed:
Create schoolsafety.gov profile

LauraUSDS committed:
Create SMEQA profile

LauraUSDS committed:
Create DITAP profile

cygov committed:
Add SMEQA press

cygov committed:
Update VA.gov press

cygov committed:
Add SchoolSafety.gov press

cygov committed:
Add DITAP press

cygov committed:
Update QPP press

cygov committed:
update MPSM press

cygov committed:
update discharge upgrade press

cygov committed:
update BB2.0 press

Added in DPC press in case you want to highlight that instead or separately.

lisac-usds committed:
wrap figure in quotes

cygov committed:
Update 0300_discharge-upgrade-tool.md

LauraUSDS committed:
Merge pull request #570 from usds/cygov-patch-2

Update VA.gov press

LauraUSDS committed:
Merge pull request #571 from usds/cygov-patch-3

Add SchoolSafety.gov press

LauraUSDS committed:
Merge pull request #572 from usds/cygov-patch-4

Add DITAP press

LauraUSDS committed:
Merge pull request #573 from usds/cygov-patch-5

Update QPP press

LauraUSDS committed:
Merge pull request #575 from usds/cygov-patch-6

update MPSM press

LauraUSDS committed:
Merge pull request #576 from usds/cygov-patch-7

update discharge upgrade press

LauraUSDS committed:
Merge pull request #569 from usds/cygov-patch-1

Add SMEQA press

LauraUSDS committed:
add image file names + caption text

LauraUSDS committed:
update image file names

LauraUSDS committed:
add image file names + caption

LauraUSDS committed:
fix image file names

LauraUSDS committed:
add images for project profiles

LauraUSDS committed:
Merge pull request #585 from usds/LauraUSDS-patch-2

add images for project profiles

cygov committed:
Update 0400_blue-button-2.md

cygov committed:
Update 0400_blue-button-2.md

LauraUSDS committed:
Update _projects/0050_smeqa.md

fix arrow in impact number (sidebar)

Co-authored-by: Lisa Chung <31673962+lisac-usds@users.noreply.github.com>

LauraUSDS committed:
Merge pull request #577 from usds/cygov-patch-8

update BB2.0 press

LauraUSDS committed:
update BB-DPC project profile with new text

edited/approved by CMS team + copy-edited by HQ

LauraUSDS committed:
Merge pull request #588 from usds/LauraUSDS-patch-2

update BB-DPC project profile with new text + images

LauraUSDS committed:
Merge pull request #563 from usds/staging-proj-updates

Staging new + updated project pages

LauraUSDS committed:
fix BBDPC live project link

change to https://dpc.cms.gov/

LauraUSDS committed:
Merge pull request #590 from usds/LauraUSDS-patch-2

fix BBDPC live project link

LauraUSDS commented:
@drew-usds @lisac-usds 

**How hard is it to change the header text for the sidebar numbers?**
I figure this is constant across all project profile pages. It currently says "The Impact" which is odd now that some project profiles contain a "The Impact" section in the body. Propose changing to "By the numbers" or something similar.

**Mostly out of curiosity, can someone explain how certain project pages are in the [/_projects folder](https://github.com/usds/website/tree/staging/_projects) but don't appear on the website?**
Specifically:
- 0600_disability-compensation-application.md
- 0800_personalized-dashboard.md
- 0900_disability-appeals-status.md
- 1400_gi-bill-comparison-tool.md
I know we don't know the decision process of why they aren't live anymore, but I'm curious if it's a fluke. Also asking how we can archive project pages, since I'm sure that'll come up and then maybe I won't have to bother y'all. ;)

CC @bernars-usa for FYI

lisac-usds commented:
Hi @LauraUSDS ! Some quick thoughts - would you please let me know whether I should implement any of these on this `staging` branch?

1. How hard is it to change the header text for the sidebar numbers?
Possibly a one-line change: 
https://github.com/usds/website/blob/ef2b53f0556c34057bc7cf99962cf1dfbea4c607/_includes/_template--project-page.html#L9-L12

2. certain project pages are in the /_projects folder but don't appear on the website?
This might be respecting a `hide` flag, eg
   - https://github.com/usds/website/blob/ef2b53f0556c34057bc7cf99962cf1dfbea4c607/_projects/0600_disability-compensation-application.md#L8-L12
   - https://github.com/usds/website/blob/ef2b53f0556c34057bc7cf99962cf1dfbea4c607/_projects/1400_gi-bill-comparison-tool.md#L8-L12

3. how we can archive project pages
Might have a few options. How about we move them into the existing [_projects-drafts](https://github.com/usds/website/tree/master/_projects-drafts) directory (and maybe rename)? Or we could create a standalone `_projects-archives` directory. Those approaches would be low level of effort.  We could also consider adding something like an `archived` flag (similar to `hide`).

lisac-usds commented on _projects/0400_blue-button-2.md:
```diff
@@ -6,15 +6,15 @@ banner_file_mobile: banner--people-projects-sm.svg
-carousel_title: "Modernizing health data access with Medicare APIs"
-carousel_summary: "We worked with CMS to build an API that enables beneficiaries with different needs to grant access to developers that can help them monitor for drug conflicts, refill prescriptions, and track progress towards desired healthcare outcomes."
-carousel_image_name: project-hhs-blue-button-card.jpg
+carousel_title: "Empowering beneficiares with their own healthcare data"
```
```suggestion
carousel_title: "Empowering beneficiaries with their own healthcare data"
```

lisac-usds commented on _projects/0400_blue-button-2.md:
```diff
@@ -6,15 +6,15 @@ banner_file_mobile: banner--people-projects-sm.svg
-carousel_summary: "We worked with CMS to build an API that enables beneficiaries with different needs to grant access to developers that can help them monitor for drug conflicts, refill prescriptions, and track progress towards desired healthcare outcomes."
-carousel_image_name: project-hhs-blue-button-card.jpg
+carousel_title: "Empowering beneficiares with their own healthcare data"
+carousel_summary: "USDS has worked with CMS to empower beneficiaries with their own health data. Leveraging CMS APIs, providers can view of their patients’ full healthcare history without patient intervention. Providers save valuable time that they can better spend talking to patients and preventing conflicts or gaps in care."
```
noticed this difference btwn the wording in `carousel_title` vs `carousel_summary`:
"...their own healthcare data" vs  "...their own health data". Would it be appropriate to use the same phrase?

lisac-usds commented on _projects/0600_ditap.md:
```diff
@@ -0,0 +1,50 @@
+
+## The Challenge
+
+Current federal procurement practices rarely provide the flexibility required to buy and deliver modern digital services. Government’s use of digital solutions has not kept pace with peoples’ expectations. As the pace of innovation has accelerated, the government’s ability to meet people’s expectations of functional digital solutions has not kept pace. To meet this demand, the Office of Federal P ...
```
the pace of people's expectations is called out in sentences 2 and 3; should we combine? Maybe we could remove sentence 2?

lisac-usds commented on _projects/0600_ditap.md:
```diff
@@ -0,0 +1,50 @@
+
+## The Solution
+
+In 2016, USDS and OFPP created the Digital IT Acquisition Professional Program (DITAP), a Federal Acquisition Certification in Contracting (FAC‑C) certification program that trains contracting professionals to execute digital service procurements by acting as business advisors to agencies seeking to buy better digital services. DITAP leverages flexibilities in the Federal Acquisition Regulation t ...
```
```suggestion
In 2016, USDS and OFPP created the Digital IT Acquisition Professional Program (DITAP), a Federal Acquisition Certification in Contracting (FAC‑C) certification program that trains contracting professionals to execute digital service procurements by acting as business advisors to agencies seeking to buy better digital services. DITAP leverages flexibilities in the Federal Acquisition Regulation to better acquire today’s technology and works seamlessly with digital service teams or acquisition innovation labs. Graduates of the DITAP program can also lead agency training workshops and provide consultations, expanding digital service procurement expertise across the government. In a [May 2018 memo](https://whitehouse.gov/wp-content/uploads/2018/05/fac_c_digital_services_05_12_18.pdf), OMB determined that beginning in FY 2022, contracting professionals assigned to digital services acquisitions above $7 million will be required to have FAC‑C‑DS certification.
```
(updated the url to include `https://` prefix; else, it'll be treated like a relative url on the usds.gov website)

LauraUSDS committed:
Update _projects/0400_blue-button-2.md

Co-authored-by: Lisa Chung <31673962+lisac-usds@users.noreply.github.com>

LauraUSDS committed:
make carousel title + summary match

Per Lisa C's suggestion: streamlined language so the phrase is "their own health data" (instead of being inconsistent)

LauraUSDS committed:
Merge pull request #592 from usds/LauraUSDS-patch-2

make carousel title + summary match

LauraUSDS committed:
remove redundant language

per Lisa C's suggestion in https://github.com/usds/website/pull/589

LauraUSDS committed:
Merge pull request #593 from usds/LauraUSDS-patch-2

remove redundant language

LauraUSDS committed:
Update _projects/0600_ditap.md

fix URL in 'solution' section

Co-authored-by: Lisa Chung <31673962+lisac-usds@users.noreply.github.com>

LauraUSDS commented:
@lisac-usds Yes, if we could make that language change for the stats sidebar (per above), that would be great!

LauraUSDS committed:
text changes

- fix URL for live project

LauraUSDS committed:
text changes

- added 'the' to 'the impact' header for consistency
- fix URL for live project
- fix capitalization in last press hit

LauraUSDS committed:
text changes

- remove redundant language in carousel summary
- remove DPC subheader in press section

LauraUSDS committed:
text changes

- clarify carousel title
- added 'the' to 'the impact' header for consistency

LauraUSDS committed:
fix memo URL

LauraUSDS committed:
text changes

- fix weird line break
- make last press hit sentence-case

LauraUSDS committed:
Merge pull request #594 from usds/LauraM-patch1

tweaks to project pages from Laura's staging site review

LauraUSDS committed:
Change stats sidebar header text

Now that several project pages include a "the impact" section, we're changing the title of the stats in the sidebar from "the impact" to "by the numbers"

LauraUSDS committed:
Merge pull request #595 from usds/LauraM-patch2

change stats sidebar header text

LauraUSDS commented:
Alright! I've proofread everything, checked formatting, caught some typos, made the sidebar header change like Lisa showed me, and.....all looks good now...so I'm going to push live. 🤞🏻

LauraUSDS commented:
Blast - I just got a 'page build failure' notification. Lisa and Drew, I'll ping y'all next week to see when you might have time to help me investigate!

lisac-usds commented:
investigating!

lisac-usds commented:
i wonder if the GitHub pages deployment failure could be due to our config referencing an older version of `jekyll-feed`. nothing jumps out at me for why we can't use the more recent version (0.13 -> 0.15). change proposed in #596.
